app/common/src/cljc/teet/util/collection.cljc

```diff
@@ -39,6 +39,13 @@
             element))
         collection))
 
+(defn find-by-id
+  "Find first element in `collection` whose `:db/id` is `id`"
+  [id collection]
+  (find-first (comp (partial = id)
+                    :db/id)
+              collection))
+
```

Comment:
Didn't end up using this after all, so if this doesn't look useful, I can remove this.